build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -28,6 +23,11 @@ steps:
 
   - template: MUX-InstallNuget-Steps.yml
 
+  - task: CmdLine@1
```

Comment:
Why did you move this?

Reply:
InstallNuget was adding stuff to the system environment and I wanted to know what it was. In particular it helpde me discover the NUGETEXETOOLPATH variable.